### DISCUSSION TITLE
does FLAML have development plan for multimodal model e.g. tabular + text + image ?

### DISCUSSION THREAD

**Discussion by wjlgatech** [microsoft/FLAML#279] [Ideas]
does FLAML have development plan for multimodal model e.g. tabular + text  or tabular+ text + image?

Autogluon by AWS recently add this feature. It would be awesome to see FLAML become competitive in this area.

**Comment by liususan091219** [comment]
Thanks for the idea and pointer.  

We are on the nlp part now. Will consider supporting multimodal for the next step. If there's an application you wish us to support, please feel free to post such request.
